deployment/cloud-config.yml

```diff
@@ -0,0 +1,27 @@
+#cloud-config
+mounts:
+- [ /dev/nvme0n1p1, /mnt/data/ ]
+
+packages:
+# These dependencies are nessecary to build Onramp Python dependencies
+- build-essential
+- python3-dev
+# For isolating Onramp build dependencies
+- python3-venv
+
+runcmd:
+# Install OSMExpress
+- curl -L https://github.com/protomaps/OSMExpress/releases/download/0.2.0/osmexpress-0.2.0-Linux.tgz | tar -xzC /usr/local/bin/ osmx
+# Clone Onramp
+- git clone https://github.com/azavea/onramp.git /opt/onramp 
```

Comment:
I'm writing up deployment instructions specific to our client's environment and I'm unsure how to handle onramp versioning. This deployment is always locked to `master`. Had you envisioned a particular strategy for this?